### PR TITLE
Setup a new release

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@Non-Contradiction this sets up the release and sets me as a new maintainer. I'll give you a bit to have a chance to dig through and make sure you're comfortable with these changes.

File: DESCRIPTION [hunk 1]
```diff
@@ -4,6 +4,7 @@ Title: Seamless Integration Between R and 'Julia'
 Version: 0.17.6
 Date: 2022-09-08
```
Comment (Non-Contradiction) [inline DESCRIPTION:5]: Date needs to be updated.

**Comment by Non-Contradiction** [comment]
Thanks very much! I'm okay with the change. The only problems are the Date and that R check seems not to be okay with multiple maintainers.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,9 +1,10 @@
 Package: JuliaCall
 Type: Package
 Title: Seamless Integration Between R and 'Julia'
-Version: 0.17.5.9000
+Version: 0.17.6
 Date: 2022-09-08
```
Comment (ChrisRackauckas) [inline DESCRIPTION:5]:
```suggestion
Date: 2024-12-05
```

**Comment by Non-Contradiction** [comment]
Most things seem okay now. And I am submitting the package to CRAN.

Some remarks for a new release: 
DESCRIPTION, NEWS, and cran-comments typically need to be updated for a release. 
And there are also some helpful checks like package `revdepcheck` for checking for reverse dependencies, `devtools::check_win_release()` and `devtools::check_win_devel()` for some further checking of the package.

**Comment by ViralBShah** [comment]
Is the test failure something that is broken on devel?

**Comment by Non-Contradiction** [comment]
> Is the test failure something that is broken on devel?

I highly suspect that the broken is due to something on the RCall side. I guess that there is some c code changing in R devel that breaks RCall.

**Comment by ViralBShah** [comment]
https://github.com/JuliaInterop/RCall.jl/issues/552

**Pull request merged**